README.md

```diff
@@ -1,68 +1,64 @@
-9wm
-====
+# 9wm
```

Comment:
That's standard Markdown [heading syntax](https://www.markdownguide.org/basic-syntax/#headings). In case need to revert, I'd suggest to use `<h2>` titles:

```
Heading 2
---------
```